server/pom.xml

```diff
@@ -164,13 +164,19 @@
               <transformers>
                 <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
                   <manifestEntries>
-                    <Main-Class>io.vertx.core.Launcher</Main-Class>
-                    <Main-Verticle>org.folio.metastorage.server.MainVerticle</Main-Verticle>
+                    <Main-Class>${vertx.launcher}</Main-Class>
+                    <Main-Verticle>${vertx.verticle}</Main-Verticle>
                     <Multi-Release>true</Multi-Release>
                   </manifestEntries>
                 </transformer>
               </transformers>
-              <artifactSet />
+              <artifactSet>
```

Comment:
I'd prefer we do not exclude these . Because it will make the fat jar without options fail upon start.